src/main/java/edu/uw/cse/testbayes/model/Probability.java

```diff
@@ -172,7 +172,12 @@ public int compareTo(Probability o) {
     }
 
     /**
-     * Returns the String representation of the Probability
```

Comment:
Just make it simpler by saying `Returns a string in the format specified by MoreObjects.toStringHelper(Object). Includes the numerator and denominator values where gcd(numerator, denominator) = 1`

Reply:
Wouldn't it be better to explain the format because otherwise, we expect them to know how ```MoreObjects.toStringHelper(Object)``` works.